src/tests/stress/pedsim/Ped_util/Agent_passive.enc

```diff
@@ -0,0 +1,92 @@
+module Agent_passive
+import Global_funs
+unsafe class Agent
```

Comment:
Could you briefly add some comments for the class `Agent` to describe what it does?